ci/00_title.md

```diff
@@ -86,27 +70,61 @@ plugins: RevealMarkdown, RevealChalkboard, RevealHighlight, RevealMath.KaTeX, Re
 
 </style>
 
-# Continuous _What_ Now...?
+# Continuous Integration (CI)
 
-<div class="blockquote">
-  Automating the integration of code changes from multiple contributors into a single software project.
-<h4>&mdash; <a href="atlassian.com/continuous-delivery/continuous-integration">Atlassian</a></h4>
+<div class="blockquote-wrapper fragment">
+  <div class="blockquote">
+     It is the practice of integrating changes to a project made by individuals into a main, shared version frequently.
+    <h4>&mdash;https://book.the-turing-way.org</h4>
+  </div>
 </div>
 
+---
+
+<!-- .slide: data-state="blue_overlay yellow_flag yellow_strip purple_half_circle_bottom purple_blob right_e_top" data-background-video="./files/steampunk-85358.mp4" data-background-video-loop data-background-video-muted="true" data-auto-animate data-auto-animate-id="title" -->
+
+
+# Motivation
+How can we implement __automatic testing__ each time we push changes to the repository? 
+
+<img src="./files/automate.jpg">
+
+
 ---
 
 <!-- .slide: data-state="blue_overlay yellow_flag logo 9" data-background="./files/reload-97640.svg" data-background-size="50%" -->
 
-## What Is It Good For? 
-- Linting
-- Automated testing
-- Security analyses
-- Send messages
-  - Slack, Discord, Matrix, Mastodon, email, ...
+## CI enables automatic testing!
```

Comment:
I don't think this is accurate, at least not according to definition above. I think "Continuous Testing" is a thing in itself, which is separate from (but related to) CI.

GitHub/GitLab actions or workflows enable automated testing.

Also, I kind of like the explanations/definitions given on [wiki](https://en.wikipedia.org/wiki/CI/CD)

Reply:
Hmm maybe, in my head Automated testing is a subset of CI. No?

Reply:
Oh I see that according to wiki continuous integration and continuous testing are separate things.. So integration is only integration and testing comes after.. Very confusing. We are teaching CI but need to talk about testing (their exercise is about testing). How would you rephrase it then?

Reply:
So, I think to me the main problem is that the way you talk about CI throughout the module does not fit the definition we give at the beginning.

Maybe one way around this is to introduce it in the first slide as: "Formally speaking, CI is defined as [XXX]. However, in daily practice, we often use this terminology in a broader sense, namely as [YYY]." 
I do think in this case we need to think about a good "definition" or description of [YYY], so that it's clear from the beginning what we're all talking about and everyone is on the same page.